OptimizelySDK.Tests/UtilsTests/ExperimentUtilsTest.cs

```diff
@@ -0,0 +1,135 @@
+﻿/* 
+ * Copyright 2019, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using NUnit.Framework;
+using OptimizelySDK.Entity;
+using OptimizelySDK.Utils;
+
+namespace OptimizelySDK.Tests.UtilsTests
+{
+    [TestFixture]
+    public class ExperimentUtilsTest
+    {
+        private ProjectConfig Config;
+
+        [SetUp]
+        public void Setup()
+        {
+            Config = ProjectConfig.Create(TestData.TypedAudienceDatafile, null, null);
+        }
+        
+        #region IsUserInExperiment Tests
+
+        [Test]
+        public void TestIsUserInExperimentReturnsTrueWithNoAudience()
+        {
+            var experiment = Config.GetExperimentFromKey("feat_with_var_test");
+            experiment.AudienceIds = new string[] { };
+            experiment.AudienceConditions = null;
+
+            Assert.True(ExperimentUtils.IsUserInExperiment(Config, experiment, null));
+        }
+
+        [Test]
+        public void TestIsUserInExperimentReturnsFalseWithAudiencesAndNoAttributes()
```

Comment:
This isn't true anymore. We do evaluate even if no attributes are passed. Please revise this to verify that audience get's evaluated with empty/null attributes. 
See this in PHP here: https://github.com/optimizely/php-sdk/blob/master/tests/UtilsTests/ValidatorTest.php#L190